model_analyzer/config/input/config_command_profile.py

```diff
@@ -461,6 +461,16 @@ def _add_run_search_configs(self):
         to the run search
         """
 
+        self._add_config(
+            ConfigField(
+                'early_exit_enable',
```

Comment:
should I include 'manual' in the name anywhere?

Reply:
No, I think this is fine